cmd/README.md

```diff
@@ -0,0 +1,618 @@
+# Precompile Generation Tutorial
```

Comment:
We should probably move this to avalanche-docs under https://docs.avax.network/subnets/customize-a-subnet.

Reply:
will do, after this gets cleaned up

Reply:
If we're going to keep it here as well, we can move it under to cmd/precompilegen/README.